scripts/postinstall.js

```diff
@@ -0,0 +1,40 @@
+const { promisify } = require('util')
+const fs = require('fs')
+const path = require('path')
+
+const copyFile = promisify(fs.copyFile)
+const mkdir = promisify(fs.mkdir)
+
+ensureEmptyDotPrisma()
+
+async function ensureEmptyDotPrisma() {
+  try {
+    const inTheNodeModules =
+      __dirname.endsWith('/node_modules/nexus-prisma/scripts') ||
```

Comment:
Should this be `/node_modules/.nexus-prisma/scripts`?

Reply:
no the scripts inside the package itself to generate the `.nexus-prisma` folder